packages/react-native/ReactCommon/react/renderer/graphics/ColorComponents.h

```diff
@@ -9,11 +9,14 @@
 
 namespace facebook::react {
 
+enum class ColorSpace { sRGB, DisplayP3 };
+
 struct ColorComponents {
   float red{0};
   float green{0};
   float blue{0};
   float alpha{0};
+  ColorSpace colorSpace{ColorSpace::sRGB};
```

Comment:
we should be able to create a `static defaultColorSpace` that we initialize based on the default color space identified by the global flag, and use that static value to initialize the structure, right?

Reply:
I wasn't able to figure out importing RCTConvert here to initialize it but I was able to do it the other way around. I'm wondering if it'd make more sense to move the global flag handling over here as well to continue to consolidate this.

Reply:
that's an option. If we can manipulate the C++ flag from Java/Kotlin (through JNI) and objectiveC and read that flag in a similar way, it is a great alternative.

C++ ColorSpace would be the source of truth and iOS/Android would just read from it.